WordPress/gutenberg
# Add link to npm release details to the release docs

## What?
Adds link to the npm release info anchor to reference at top of docs

## Why?
The npm details are a long way down the page, so saves people new to release process having to scroll through full doc to find the details.

## How?
Added a markdown anchor link

## Testing Instructions
View edited file and make sure link to NPM publishing details at top of page works as expected
